app/controllers/checks_average_by_month_controller.rb

```diff
@@ -0,0 +1,69 @@
+class ChecksAverageByMonthController < ApplicationController
+  helper_method :show_months, :calculate_average_of_times
+
+  def index
+    @checks_average_by_month = Attendance.all
+    @employees = Employee.all
+  end
+
+  def show_months
+    attendances = Attendance.all
+    employees = Employee.all
+    
+    months = []
+    m=[]
```

Comment:
When you write variables, it is recommended to give them a meaningful name and a length greater than three letters. This allows other developers, or yourself in the future, to know what the function of that variable is.

Reply:
Understood, we take it as an auxiliary variable but as you say, we better specify it, thanks for the observation